react-js/styling-react.md

```diff
@@ -0,0 +1,250 @@
+# Styling in React
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Guided Practice: 20 mins.
+
+### Prerequisites
+- [CSS Lesson](/css/css)
+- [Bootstrap Lesson](/bootstrap/bootstrap)
+- [React Lesson](react)
+
+
+### Motivation
+
+Styling has been one of the building blocks of web development. Cascading Style Sheets is a style sheet language used for describing the presentation of a document written in a markup language like HTML. React only provides a virtual DOM, whereas the presentation part still lies with CSS.
+
+### Objectives
+
+- Styling in JS Framework.
+- Inline styling and using external Stylesheet.
+- Bootstrap in React.	 
+
+### Specific Things To Learn
+
+- Defining styles as objects.
+- Importing stylesheets into react application.
+
+### Materials
+
+- [Video: Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external CSS, internal and CSS modules.
```

Comment:
are we using "CSS Modules" anywhere now?  I don't see it.

Reply:
Yes it is similar to External CSS. But it is important to know, in case of multiple CSS file.